repos/newtonsoft-json.proj

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <PreferredHash>e43dae94c26f0c30e9095327a3a9eac87193923d</PreferredHash>
+    <PreferredHash>cac0690ad133c5e166ce5642dc71175791404fad</PreferredHash>
     <SourceDirectory>Newtonsoft.Json</SourceDirectory>
   </PropertyGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
 
+  <ItemGroup>
+    <EnvironmentVariables Include="LibraryFrameworks=netstandard1.0%3bnetstandard1.3%3bnetstandard2.0" />
```

Comment:
Please add a comment here about how we're doing this to avoid the semicolons getting parsed by the shell.  Also one nitpick - please capitalize the B in `%3B` so it stands out from the TFMs more.